core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -395,6 +395,74 @@ public enum Property {
           + "indefinitely. Default is 0 to block indefinitely. Only valid when tserver available "
           + "threshold is set greater than 0. Added with version 1.10",
       "1.10.0"),
+  // properties that are specific to scan server behavior
+  @Experimental
+  SSERV_PREFIX("sserver.", null, PropertyType.PREFIX,
+      "Properties in this category affect the behavior of the scan servers", "2.1.0"),
+  @Experimental
+  SSERV_DATACACHE_SIZE("sserver.cache.data.size", "10%", PropertyType.MEMORY,
+      "Specifies the size of the cache for RFile data blocks on each scan server.", "2.1.0"),
+  @Experimental
+  SSERV_INDEXCACHE_SIZE("sserver.cache.index.size", "25%", PropertyType.MEMORY,
+      "Specifies the size of the cache for RFile index blocks on each scan server.", "2.1.0"),
+  @Experimental
+  SSERV_SUMMARYCACHE_SIZE("sserver.cache.summary.size", "10%", PropertyType.MEMORY,
+      "Specifies the size of the cache for summary data on each scan server.", "2.1.0"),
+  @Experimental
+  SSERV_DEFAULT_BLOCKSIZE("sserver.default.blocksize", "1M", PropertyType.BYTES,
+      "Specifies a default blocksize for the scan server caches", "2.1.0"),
+  @Experimental
+  SSERV_CACHED_TABLET_METADATA_EXPIRATION("sserver.cache.metadata.expiration", "5m",
+      PropertyType.TIMEDURATION, "The time after which cached tablet metadata will be refreshed.",
+      "2.1.0"),
+  @Experimental
+  SSERV_PORTSEARCH("sserver.port.search", "true", PropertyType.BOOLEAN,
+      "if the ports above are in use, search higher ports until one is available", "2.1.0"),
+  @Experimental
+  SSERV_CLIENTPORT("sserver.port.client", "9996", PropertyType.PORT,
+      "The port used for handling client connections on the tablet servers", "2.1.0"),
+  @Experimental
+  SSERV_MAX_MESSAGE_SIZE("sserver.server.message.size.max", "1G", PropertyType.BYTES,
+      "The maximum size of a message that can be sent to a scan server.", "2.1.0"),
+  @Experimental
+  SSERV_MINTHREADS("sserver.server.threads.minimum", "2", PropertyType.COUNT,
+      "The minimum number of threads to use to handle incoming requests.", "2.1.0"),
+  @Experimental
+  SSERV_MINTHREADS_TIMEOUT("sserver.server.threads.timeout", "10s", PropertyType.TIMEDURATION,
```

Comment:
```suggestion
  SSERV_MINTHREADS_TIMEOUT("sserver.server.threads.timeout", "0s", PropertyType.TIMEDURATION,
```

Make this consistent w/ tserver setting as mentioned in #2745